flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/calcite/FlinkTypeFactoryTest.scala

```diff
@@ -0,0 +1,47 @@
+package org.apache.flink.table.calcite
+
+import org.apache.flink.api.scala.typeutils.Types
+
+import org.apache.calcite.sql.`type`.SqlTypeName
+import org.junit.Assert.{assertFalse, assertTrue}
+import org.junit.Test
+
+/** Test cases for [[FlinkTypeFactory]]. */
+class FlinkTypeFactoryTest {
+
+  @Test
+  def testCanonizeType(): Unit = {
```

Comment:
I find the test does not cover the changes. I revert the changes and run the method, all tests are success.

Reply:
Thanks for the check, i have updated the test cases.